client/src/redux/sagas/allocations.sagas.js

```diff
@@ -278,11 +278,46 @@ export function* addUser(action) {
     console.log(error);
   }
 }
+
+export const allocationsTeamSelector = state => state.allocations.teams;
+
 export function* removeUser(action) {
   try {
+    yield put({
+      type: 'ALLOCATION_OPERATION_REMOVE_USER_STATUS',
+      payload: {
+        removingUserOperation: {
```

Comment:
Based on how we've been handling reducer state updates in the past, should we move the `removingUserOperation` portion of this payload to the reducer? That way it's clear what the reducer is doing to the state

Reply:
Yes,  so you are saying change the type here to something like "ALLOCATION_OPERATION_REMOVE_USER_START" (Note the START bit at the end and then only the `username` would be in the payload.  Can you confirm if I am understanding correctly?

Reply:
Yes, or you can even keep it as `_STATUS` or `_UPDATE` and pass `loading` and `error` as well

Reply:
> @rstijerina , I think i understand all your comments (but would like to confirm [#474 (comment)](https://github.com/TACC/Core-Portal/pull/474#discussion_r700476577)).
> 
> Is it okay if I merge as-is. I would then pick up the `Extend saga/reducer so that added user is added to list` todo item for the 737 work and address these comments then as they sorta relate. cool?

Yep 👍